### ISSUE TITLE
Publish TEKGEN

### ISSUE THREAD

**Issue by nicir** [google-research-datasets/KELM-corpus#2]
Hello guys, 

currently I am looking for a method to generate natural sentences from KG triples for a research project. I consider TEKGEN as a great tool to do so. Do you plan to publish TEKGEN in near future? 

Greetings

**Comment by oagarwal** [comment]
If you have sets of KG triples already available, a pretrained T5 model can be finetuned on first the released TEKGEN Corpus and then the WebNLG data (https://webnlg-challenge.loria.fr/challenge_2017/#data) using the hyperparameters in the paper to generate natural language sentences. Would that work for your use case? Currently, we do not plan to release the finetuned checkpoints.

**Comment by nicir** [comment]
Unfortunately that won't work for us as I work at a university where we haven't got lots of computional resources to develop the model. 
Our idea is to finetune language models with subject-specific knowledge that we want to extract from certain ontologies. That's why we were wondering if we could use your model. But probably we'll have to search for an easier way then...

**Comment by salym** [comment]
I agree that publishing the pipeline would be a great share.

**Comment by salym** [comment]
I agree that publishing the pipeline would be a great share. It would fit perfectly our e-tutor app project. It's a family project led by my 10yo kid and we don't have any sufficient GPU resources.

**Comment by gilljon** [comment]
Any plans to release the fine-tuned T5 model?
